src/inmanta/agent/handler.py

```diff
@@ -265,7 +265,7 @@ def changed(self) -> bool:
     def change(self) -> const.Change:
         return self._change
 
-    def add_change(self, name: str, desired: typing.Any, current: typing.Any = None) -> None:
+    def add_change(self, name: str, desired: object, current: object = None) -> None:
```

Comment:
Could you attach the mypy diff as well?

Reply:
The diff is empty, which makes sense because of the usage of `Any`.